deploy/yamls/zfssnapshot-crd.yaml

```diff
@@ -35,20 +46,26 @@ spec:
         metadata:
           type: object
         spec:
-          description: VolumeInfo contains the volume related info for all types of
-            volumes in ZFSVolume
+          description: VolumeInfo defines ZFS volume parameters for all modes in which
+            ZFS volumes can be created like - ZFS volume with filesystem, ZFS Volume
+            exposed as zfs or ZFS volume exposed as raw block device. Some of the
+            parameters can be only set during creation time (as specified in the details
+            of the parameter), and a few are editable. In case of Cloned volumes,
+            the parameters are assigned the same values as the source volume.
           properties:
             capacity:
               description: Capacity of the volume
               minLength: 1
               type: string
             compression:
               description: Controls the compression algorithm used for this dataset.
-                Compression specifies if the it should enabled on the zvol. Setting
-                compression to on indicates that the current default compression algorithm
-                should be used. The current default compression algorithm is either
-                lzjb or, if the lz4_compress feature is enabled, lz4. Changing this
-                property affects only newly-written data.
+                Compression specifies if it should be enabled on the zvol/dataset.
+                Setting compression to "on" indicates that the current default compression
+                algorithm should be used. The current default compression algorithm
+                is either lzjb or, if the lz4_compress feature is enabled, lz4. Changing
+                this property affects only newly-written data. By default compression
+                is set to "off". This attribute is editable, we can update this attribute
+                to the desired compression property later on once volume has provisioned.
```

Comment:
n_: The sentences should be framed using a third-person tone. Avoid using any kind of pronouns like he/she/we/it. 

This description could be changed to the following:
```
Compression specifies the block-level compression algorithm to be applied to the ZFS Volume. 
The value "on" indicates ZFS to use the default compression algorithm. The default compression
 algorithm used by ZFS will be either lzjb or, if the lz4_compress feature is enabled, lz4.
 
Compression property can be edited after the volume has been created.  The change will only
be applied to the newly-written data. For instance, if the Volume was created with "off" and 
the next day the compression was modified to "on", the data written prior to setting "on" will
not be compressed. 

Default Value: Off. 
```